Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -259,16 +262,75 @@ def get_current_user(self):
         res = self._get_sdk().users.get_current()
         return res
 
-    def get_user_id(self, username):
+    def get_user(self, username):
         res = self._get_sdk().users.get_by_username(username)["users"]
         if not res:
             raise Exception("No user found with username {0}.".format(username))
-        return res[0]["userUid"]
+        return res[0]
+
+    def create_user(self, org_name, username, email):
+        org_uid = self._get_org_id(org_name)
+        response = self._get_sdk().users.create_user(org_uid, username, email)
+        return json.loads(response.text)
+
+    def block_user(self, username):
+        user_id = self._get_legacy_user_id(username)
+        self._get_sdk().users.block(user_id)
+        return user_id
+
+    def unblock_user(self, username):
+        user_id = self._get_legacy_user_id(username)
+        self._get_sdk().users.unblock(user_id)
+        return user_id
+
+    def deactivate_user(self, username):
+        user_id = self._get_legacy_user_id(username)
+        self._get_sdk().users.deactivate(user_id)
+        return user_id
+
+    def reactivate_user(self, username):
+        user_id = self._get_legacy_user_id(username)
+        self._get_sdk().users.reactivate(user_id)
+        return user_id
+
+    def get_org(self, org_name):
+        org_pages = self._get_sdk().orgs.get_all()
+        for org_page in org_pages:
+            orgs = org_page["orgs"]
+            for org in orgs:
+                if org.get("orgName") == org_name:
+                    return org
+        raise Exception("No org found with name {0}.".format(org_name))
```

Comment:
If you've created a Code42UserIDNotFoundError, there should also probably be a custom error for this